api/src/main/java/org/openmrs/module/cohort/CohortAttribute.java

```diff
@@ -97,7 +97,6 @@ public Integer getId() {
 	@Override
 	public void setId(Integer arg0) {
 		setCohortAttributeId(arg0);
-		
```

Comment:
Very minor, but maybe we can use this opportunity to rename that variable from `arg0`?